hazelcast/src/test/java/com/hazelcast/jet/core/MemberReconnectionStressTest.java

```diff
@@ -116,5 +120,14 @@ public void test() {
             }
             sleepMillis(100);
         }
+        //Test finished, close the threads
+        try {
+            terminated.set(true);
+
+            connectionThread.join();
+            newJobThread.join();
+        } catch (Exception e) {
+            throw new RuntimeException(e);
+        }
```

Comment:
nitpick: We can just add throws Exception to the test.

Reply:
Oj